public_html/wp-content/plugins/wc-fonts/wc-fonts.php

```diff
@@ -243,7 +291,7 @@ public function validate_options( $input ) {
 			$url = wp_parse_url( $input['font-awesome-url'] );
 
 			if ( isset( $url['host'] ) && isset( $url['path'] ) ) {
-				$valid_hostname  = 'maxcdn.bootstrapcdn.com' === $url['host'];
+				$valid_hostname  = in_array( $url['host'], [ 'maxcdn.bootstrapcdn.com', 'stackpath.bootstrapcdn.com' ] );
```

Comment:
Looks like Bootstrap CDN uses `stackpath.bootstrapcdn.com` by default now, though maxcdn will still work. If someone wants to use 4.x of Font Awesome, they'll likely get the stackpath link which would be stripped on save currently.